.github/workflows/file-size-impact/report-file-size-impact.js

```diff
@@ -0,0 +1,14 @@
+const { reportFileSizeImpact, readGithubWorkflowEnv } = require("@jsenv/file-size-impact");
```

Comment:
You could use

```js
import { reportFileSizeImpact, readGithubWorkflowEnv } from "@jsenv/file-size-impact"
```

Either by adding
```json
{
  "type": "module"
}
```

In `.github/workflows/file-size-impact/package.json` or by renaming `report-file-size-impact.js` into `report-file-size-impact.mjs`.

Reply:
Done!